source/_references/sourceimages.md

```diff
@@ -141,9 +141,28 @@ $isDeleted = $client->deleteSourceImagesWithBinaryHash('03b3e8a0bdd76ef55c021066
 var_dump($isDeleted);
 ```
 
+## Copy a source image to another organization
+
+To copy a source image to another organization (for example for copying images from a production to a test environment),
+you can do a `COPY` request to  `/sourceimages/{org}/{hash}` and send the `Destination` header with the destination organization.
+If you send a `Overwrite: F` header with it, it won't copy it, if it already exists at the destination.
+Your API user needs to have read permissions on the source organization and write permissions on the destination organization. 
+The [memberships endpoint](memberships.html) has methods to give an API user access to more organizations.
+
+
+Returns a 201 http code, if the image was newly created at the destination.
+Returns a 204 http code, if the image was overwritten at the destination.
+Returns a 412 http code, if `Overwrite: F` was sent and the destination exists already.
+Returns a 404 http code, if the source image was not found.
```

Comment:
Do we return a `400` if the destination organization does not exist?

Reply:
Do we return a `403` if the user has no write permissions on `Destination` org?

Reply:
Both added and the rest is adjusted too, thank